.gitignore

```diff
@@ -3,3 +3,8 @@
 *.lo
 *.a
 *.la
+
+backtrace-supported\.h

```

Comment:
Does cmake support building in a separate directory, so that the sources can be read-only?  If so, let's not modify .gitignore here.

Reply:
I'm not really that familiar with autotools but I've built this library using `./configure` and `make` and it generated headers in source directory, as well as .a files. So I thought having .a and not .h in gitignore was a bug, not a feature. If I'm wrong I'll remove this line.

Reply:
The expectation is that people will run the equivalent of

```
mkdir x
cd x
../libbacktrace/configure
```

rather than running `.configure`.  It's OK to run `./configure`, but it shouldn't be necessary, and we need to make sure that running `configure` from a different directory works without modifying the source directory.